build_tools/third_party/s3_management/manage.py

```diff
@@ -289,10 +287,10 @@ def to_simple_package_html(
             if any(obj.key.endswith(x) for x in ("networkx-3.3-py3-none-any.whl", "networkx-3.4.2-py3-none-any.whl")):
                 attributes += ' data-requires-python="&gt;=3.10"'
 
-            stripped_key = obj.key.removeprefix(STRIP_PREFIX) if STRIP_PREFIX else obj.key
+            stripped_key = obj.key.split("/")[-1]
 
             out.append(
-                f'    <a href="/{stripped_key}{maybe_fragment}"{attributes}>{path.basename(obj.key).replace("%2B","+")}</a><br/>'
+                f'    <a href="../{stripped_key}{maybe_fragment}"{attributes}>{path.basename(obj.key).replace("%2B","+")}</a><br/>'
             )
```

Comment:
> One can install from https://rocm.prereleases.amd.com/whl-staging/gfx950-dcgpu/ to test. The index was updated manually with the updated script.

Thanks. I'm comparing
* https://rocm.nightlies.amd.com/v2-staging/gfx950-dcgpu/torch
* https://rocm.prereleases.amd.com/whl-staging/gfx950-dcgpu/torch

Indeed, this changes links from absolute like `<a href="/v2-staging/gfx950-dcgpu/torch-2.10.0a0` to relative like `<a href="../torch-2.10.0a0`